src/components/common/PopupModal.tsx

```diff
@@ -27,9 +28,80 @@ const PopupModal: React.FC<InfoModalProps> = ({
   title,
   message,
   example,
+  type = "username",
 }) => {
   const { t } = useTranslation();
 
+  const fieldConfig: any = {
```

Comment:
This is not the correct way to implement it. The expected approach is to use language constants. Since this is a common component, you should not write code here; you are only responsible for designing it.